src/magic.h

```diff
@@ -227,6 +227,8 @@ class spell_type
 
         // base amount of time to cast the spell in moves
         int base_casting_time;
+        // If spell is to summon a vehicle, the vproto_id of the vehicle
+        std::string vehicle_id;
```

Comment:
So instead of adding a new member, just reuse effect_str. that's what that particular member is for; miscellaneous string information specific to the effect type.